src/env.cc

```diff
@@ -354,6 +361,13 @@ void Environment::Start(bool start_profiler_idle_notifier) {
   uv_key_set(&thread_local_env, this);
 }
 
+void Environment::ExitEnv() {
+  set_can_call_into_js(false);
+  GetAsyncRequest()->Stop();
+  if (isolate_ != nullptr)
```

Comment:
Is this condition ever true?

Reply:
Sorry, yes, I meant the opposite. `isolate_` should not be able to change, because it’s declared as `const` inside the `Environment` class.